plugins/civo/civo.go

```diff
@@ -0,0 +1,25 @@
+package civo
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func CivoCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Civo CLI", // TODO: Check if this is correct
+		Runs:      []string{"civo"},
+		DocsURL:   sdk.URL("https://civo.com/docs/cli"), // TODO: Replace with actual URL
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
```

Comment:
Let's also opt out of authentication for `apikey save` and when `--config` is passed.

Reply:
@hculea Thank you for you guidance. I made changes in files.

Reply:
Looking through https://www.civo.com/docs/overview/civo-cli#commands-and-help I can see a few other commands that I don't think should require authentication, such as `civo update`. Can you please have a look over the rest as well? 😄 Would love to keep this list as precise as possible.